bundles/org.openhab.binding.mielecloud/src/main/resources/OH-INF/thing/channelTypes.xml

```diff
@@ -246,6 +272,10 @@
 		<label>@text/channel-type.mielecloud.temperature_target.label</label>
 		<description>@text/channel-type.mielecloud.temperature_target.description</description>
 		<category>Number:Temperature</category>
+		<tags>
+			<tag>Measurement</tag>
```

Comment:
```suggestion
			<tag>Setpoint</tag>
```

Reply:
Is it read only? I have seen other things which expose a read only view of their setpoint, but since the Setpoint point class implies controllability we had to use Measurement instead.

Reply:
temperature_target can be set by the user, it is controlleable. When this is changed the fridge/freezer will decrease / increase cooling so temperature_current moves closer to temperature_target. So setpoint seems correct to me for temperature_target.

Reply:
But this channel type IS read-only. So the fundamental (philosophical) question is: shall read-only "setpoint"s be tagged as `SetPoint` or as `Measurement` ?? .. until now my working assumption has been the latter (which is more compliant with the rules) .. however the latter also has some merit. In any case it would be good to have a clear consensus on the "correct" interpretation since it will impact many other bindings too. => Therefore I call on our panel of experts @jlaur @rkoshak @lsiepel @jimtng to cast your votes please.

Reply:
> When this is changed the fridge/freezer will decrease / increase cooling

Looking at the code, the fridge / freezer things don't use `temperature_target` channeltype. They use a different `fridge_temperature_target` channeltype.

The things that use _this_ channel type are oven, washer/dryer, etc.

I agree with @andrewfg, readonly channels needs to be `Measurement`, not `Setpoint`.

Reply:
Can we send a command to this channel? yes -> setpoint, no -> maybe `Status` if `Measurement` doesn't make sense.

Reply:
Why does it matter if we can send a command? It's still a setpoint.

Reply:
If `Setpoint` is a child of `Control` then I would vote for @jimtng whereas if not I would vote for @jlaur .. so lets see the answer:

https://github.com/openhab/openhab-core/blob/12a10b9fa4669f409ea364dd9c575d700fe9cf3c/bundles/org.openhab.core.semantics/model/SemanticTags.csv#L38

Reply:
The semantic model is trying to differentiate between write channel vs read channel.
Generally, write = Control or Setpoint, read = Status or Measurement

So we now encounter this case where it's a "setpoint" that is read only. I am confused but curious, how is it set?

Reply:
> I am confused but curious, how is it set?

It is set by the user outside of OH. e.g. via an App or a knob on the front of the device (in the case of an oven (say) you may want the SP to be only settable by physically present persons.

Reply:
OK in that case I'll update my interpretation of `Setpoint`.

Reply:
@jimtng do we -- therefore -- need to adjust either a) the semantic model, b) the UI, or c) the docs .. to match that conclusion?

Reply:
I am confused with the model now. Best ask @rkoshak's opinion maybe?

Reply:
Perhaps @mherwege can comment if allowing read only SetPoints has any impact on the UI?

Reply:
Note: in https://github.com/openhab/openhab-core/pull/4750 we applied the following description texts. So I hope that is still considered Ok?

```
Point,Measurement,,Measurement,,Read only analog data point from a sensor
Point,Setpoint,,Setpoint,,Read-Write analog data point for a controller
```

Reply:
Applied `Setpoint` anyway